### DISCUSSION TITLE
ESP32 Arduino Core verstion

### DISCUSSION THREAD

**Discussion by pink0D** [h2zero/NimBLE-Arduino#748] [Q&A] [answered]
Hi
Didn't find the information in readme, so posting this question.
What versions of ESP32 Arduino Core are supported? Is the newer 3.x supported ?
(also found one closed issue https://github.com/h2zero/NimBLE-Arduino/issues/686, but it's still unclear if 1.4.2 is stable for Arduino Core 3.x)

**Comment by h2zero** [comment, marked as answer]
You can use any core version, they all should work fine.
